stackslib/src/net/http.rs

```diff
@@ -155,6 +158,7 @@ lazy_static! {
             *STANDARD_PRINCIPAL_REGEX_STRING, *CONTRACT_NAME_REGEX_STRING
         )).unwrap();
     static ref PATH_OPTIONS_WILDCARD: Regex = Regex::new("^/v2/.{0,4096}$").unwrap();
+    static ref PATH_POST_BLOCK_PROPOSAL: Regex = Regex::new(&format!("^{PATH_STR_POST_BLOCK_PROPOSAL}$")).unwrap();
```

Comment:
Small nit: can you just get rid of `PATH_STR_POST_BLOCK_PROPOSAL` and put it here directly?  We've been bitten before by not having the complete regex in one place in the code.